R/check.R

```diff
@@ -175,36 +184,36 @@ coerce_dt <- function(
 
   if ((length(required_cols) > 0)) {     # if we have required columns ...
     if (!is.character(required_cols)) {  # ... check they are check-able
-      stop("`required_cols` must be a character vector")
+      cli::cli_abort("`required_cols` must be a character vector")
     }
     # check that all required columns are present
     if (!all(required_cols %in% colnames(dt))) {
-      stop(
-        msg_required,
-        toString(required_cols[!(required_cols %in% colnames(dt))]),
-        " but are not present among ",
-        toString(colnames(dt)),
-        "\n(all `required_cols`: ",
-        toString(required_cols),
-        ")"
+      cli::cli_abort(
+        c(
+          "{msg_required}",
```

Comment:
```suggestion
          msg_required,
```
I think this wants to remain the variable, so that we can pass in a glue-format string here?

Reply:
should double check the behavior of that, however.